hbase_regionserver/_metrics.yml

```diff
@@ -41,14 +41,17 @@
   attribute: sink.appliedOps
   metric_type: gauge
   alias: hbase.regionserver.replication.sink.applied_ops
+  description: Number of WAL entries applied on replication sink.
 - bean: Hadoop:service=HBase,name=RegionServer,sub=Replication
```

Comment:
would you mind adding `unit_name: millisecond` here too?  `_gen.rb` supports `unit_name` in `_metrics.yml`.